### DISCUSSION TITLE
TweenCancelBehaviour: difference between Kill and Kill and Cancel Await

### DISCUSSION THREAD

**Discussion by brantkings** [Cysharp/UniTask#480] [Q&A]
Hey! There's a lot of options for the behaviour when cancelling a tween from DoTween used an an UniTask. What is the difference between them? I couldn't find documentation on this.

Here are the options:

`
        Kill,
        KillWithCompleteCallback,
        Complete,
        CompleteWithSequenceCallback,
        CancelAwait,
        KillAndCancelAwait,
        KillWithCompleteCallbackAndCancelAwait,
        CompleteAndCancelAwait,
        CompleteWithSequenceCallbackAndCancelAwait
`

**Comment by SherryOever18** [comment]
https://github.com/Cysharp/UniTask/issues/526
You can find answer here.
